products/amazon-documentdb.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Amazon DocumentDB
+category: service
+tags: amazon database
+iconSlug: amazondocumentdb
+permalink: /amazon-documentdb
+releasePolicyLink: https://docs.aws.amazon.com/documentdb/latest/developerguide/compatibility.html
+releaseDateColumn: true
+releaseColumn: true
```

Comment:
Not sure the `Latest` column is needed if we don't document the engine patch version. WDYT ?

Reply:
Agreed